src/xcode/ENA/ENA/Source/Services/Risk/Calculation/RiskCalculation.swift

```diff
@@ -77,10 +77,10 @@ enum RiskCalculation {
 		}
 
 		// Precondition 1 - Exposure Notifications must be turned on
-		guard preconditions.isGood else {
-			// This overrides all other levels
-			return .success(.inactive)
-		}
+//		guard preconditions.isGood else {
```

Comment:
@inf2381 should we comment code here?

Reply:
@aleksandr-tikhonov I think we should revert these changes and fix the root cause in a separate PR